pkg/printers/internalversion/printers.go

```diff
@@ -2016,6 +2039,66 @@ func printVolumeAttachmentList(list *storage.VolumeAttachmentList, options print
 	return rows, nil
 }
 
+func printFlowSchema(obj *flowcontrol.FlowSchema, options printers.PrintOptions) ([]metav1beta1.TableRow, error) {
```

Comment:
Note to self: we should be able to mark fields as important in the schema and generate functions like this one. :/